tests/common/include/tests/common/tools_propagator.inl

```diff
@@ -131,6 +238,10 @@ TEST_P(PropagatorWithRkStepper, propagator_rk_stepper) {
     // Set origin position of tracks
     const point3 ori{0., 0., 0.};
 
+    auto actor_states =
```

Comment:
Could you define `actor_chain_t::state` in actor_chain struct so you don't have to write down aggregation twice?

Reply:
I am not sure I understand. I still have to call the constructors of the states, right?

Reply:
Right